CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+1. **Socialize the idea**: Before starting the review process, it's helpful to gauge interest from the community. Ideally, the idea would have come up in a discussion in another issue or in [chat][] and the consensus was to start an evolution proposal.
+1. **Develop the proposal**: open a Pull Request with a rough sketch of the proposal using the [proposal template](template.md) in the [_drafts](https://github.com/hubotio/evolution/new/master/_drafts) directory. Prototyping an implementation and its uses along with the proposal is encouraged, because it helps ensure both technical feasibility of the proposal as well as validating that the proposal solves the problems it is meant to solve.
+1. **Request a review**: When you are ready for feedback from the Hubot community, change the frontmatter in your proposal from `state: In Progress` to `state: In Review`.
+1. **Address feedback**: be responsive to questions and feedback about the proposal. When the proposal is sufficiently detailed and clear, and addresses feedback from earlier discussions of the idea, the `accepted` label will be added and a maintainer will be assigned to manage the review.
+
+## Review process
+
+When you change the frontmatter in your proposal from `state: In Progress` to `state: In Review`, a message will be posted in the [#evolution channel in chat][chat] requesting feedback from the community.
+
+Reviews should usually be completed in a single week, but can run longer for particularly large or complex proposals. To avoid delays, it is important that the proposal authors be available to answer questions, address feedback, and clarify their intent during the review period.
+
+After feedback from the community has been gathered, the maintainers will make a decision on whether to `accept` or `reject` the proposal.
+
+Once a proposal has been accepted, the file should be renamed to `proposals/###-feature-name.md`, where `###` is the Pull Request number. It is usually up to the proposal author to implement it or discuss with the maintainers who will lead the implementation.
+
+Once a proposal has been implemented, the Pull Request will be merged.
+
+## Proposal states
+
+A proposal can be in one of several states:
+
+* **In Progress**: The authors are still working on the proposal and are not yet ready for review.
+* **In Review**: The proposal is ready for review and comments from the community.
+* **Withdrawn**: The proposal has been withdrawn by the original submitter.
+* **Deferred**: Consideration of the proposal has been deferred because it does not meet the [goals of the upcoming release](README.md). Deferred proposals will be reconsidered when scoping the next major release.
```

Comment:
I really ❤️ this state. I think there has always been some tension when adding new features, where I haven't wanted to say no to them, but now just didn't feel like the right time to do them. Deferred feels right in this context, as it doesn't mean no, just not now, with a concrete way to revisit them after the next major release.